common/app/common/Preload.scala

```diff
@@ -7,7 +7,11 @@ import experiments.ActiveExperiments
 
 object Preload {
 
-  def commercialBundleName(implicit request: RequestHeader): String = "graun.commercial.js"
+  def commercialBundleName(implicit request: RequestHeader): String = "commercial/graun.commercial.dcr.js"
+
+  // TODO: start using it
+  def commercialStandaloneBundleName(implicit request: RequestHeader): String =
```

Comment:
this doesn't start preloading this file yet, is that right?

Reply:
No, it doesn’t, but should it?

I’m thinking that if the request is available as an implicit, we could use the participation in the test group to choose which bundle to serve.